tooling/cli/dist/index.bin.js

```diff
@@ -1 +0,0 @@
-#!/usr/bin/env node
```

Comment:
Don't delete , just unstaged in your commit.

Reply:
@ri7nz i can't revert this one

Reply:
Okay, nope. 

You can follow this guide:

* run `git reset HEAD^` 3 times.
* re run `git add` 
* ru run `git commit`

Reply:
https://github.com/bl0cknumber/chakra-icons/blob/8af8d48e40ad83fb5ca5ed486ad6e4137c22c639/.gitignore#L59

Reply:
See this https://github.com/kodingdotninja/chakra-icons/issues/33 why this files `ignored` but existed